File: src/domain/entities/geocode.entity.ts
export class GeocodeEntity {
    constructor(
        public city: string,
        public latitude: number,
        public longitude: number,
    ){}

    public static fromObject(object: {[key: string] : any}) : GeocodeEntity {
        const { city, latitude, longitude } = object;
        return new GeocodeEntity(
            city,
            latitude,
            longitude
        );
    }
}

File: python/app/domain/geocode_entity.py
class GeocodeEntity:
    def __init__(self, city: str, latitude: float, longitude: float):
        self.city = city
        self.latitude = latitude
        self.longitude = longitude

    @staticmethod
    def from_dict(data: dict) -> 'GeocodeEntity':
        return GeocodeEntity(
            city=data.get('city'),
            latitude=data.get('latitude'),
            longitude=data.get('longitude')
        )
